astropy/visualization/stretch.py

```diff
@@ -4,9 +4,12 @@
 Classes that deal with stretching, i.e. mapping a range of [0:1] values onto
 another set of [0:1] values with a transformation.
 """
+import warnings
 
 import numpy as np
 
+from astropy.utils.exceptions import AstropyDeprecationWarning
```

Comment:
```suggestion
from astropy.utils.decorators import deprecated
```

Reply:
```suggestion
from astropy.utils.exceptions import deprecated_attribute
```

Reply:
Thanks, @eerovaher!  Somehow I wasn't aware of this.  Very useful!  (it's `astropy.utils.decorators.deprecated_attribute`)

Reply:
I forgot to update the module in my suggestion, but it's good that you managed to find `deprecated_attribute` regardless.

Reply:
Thanks, @eerovaher ! I was looking at the module and somehow it didn't click in my brain that Larry was trying to deprecate attribute, not property. 😅